pennylane/capture/capture_qnode.py

```diff
@@ -61,7 +61,7 @@ def _get_qnode_prim():
     qnode_prim.multiple_results = True
 
     @qnode_prim.def_impl
-    def _(*args, shots, device, qnode_kwargs, qfunc_jaxpr):
+    def _(*args, qnode, shots, device, qnode_kwargs, qfunc_jaxpr):
```

Comment:
This is not being used here, but I'm assuming it will be used in Catalyst?

Reply:
Yep.  The corresponding primitive in catalyst `catalyst.jax_primitives.func_p` requires the qnode to be passed to the `"fn"` keyword argument.